extensions/google-calender/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "google-calender",
+  "title": "Google Calender",
+  "description": "Search and Create sharable link to registrate!",
+  "icon": "google-calender-icon.png",
+  "author": "nagauta",
+  "categories": [
+    "Productivity",
+    "Applications",
+    "Communication"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "generate-event-reg-link",
+      "title": "Create Sharable Event Link",
+      "subtitle": "Google Calender",
+      "description": "Easy to generate evet link to register for Google Calender!",
+      "mode": "view"
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "^1.54.1",
```

Comment:
Please update to the latest Raycast API.

Reply:
I updated  to the latest Raycast API now.